updates/v2.0.2/seed_ru_states.php

```diff
@@ -0,0 +1,121 @@
+            ['code' => '44', 'name' => 'Костромская область'],
+            ['code' => '45', 'name' => 'Курганская область'],
+            ['code' => '46', 'name' => 'Курская область'],
+            ['code' => '47', 'name' => 'Ленинградская область'],
+            ['code' => '48', 'name' => 'Липецкая область'],
+            ['code' => '49', 'name' => 'Магаданская область'],
+            ['code' => '50', 'name' => 'Московская область'],
+            ['code' => '51', 'name' => 'Мурманская область'],
+            ['code' => '52', 'name' => 'Нижегородская область'],
+            ['code' => '53', 'name' => 'Новгородская область'],
+            ['code' => '54', 'name' => 'Новосибирская область'],
+            ['code' => '55', 'name' => 'Омская область'],
+            ['code' => '56', 'name' => 'Оренбургская область'],
+            ['code' => '57', 'name' => 'Орловская область'],
+            ['code' => '58', 'name' => 'Пензенская область'],
+            ['code' => '59', 'name' => 'Пермский край'],
+            ['code' => '60', 'name' => 'Псковская область'],
+            ['code' => '61', 'name' => 'Ростовская область'],
+            ['code' => '62', 'name' => 'Рязанская область'],
+            ['code' => '63', 'name' => 'Самарская область'],
+            ['code' => '64', 'name' => 'Саратовская область'],
+            ['code' => '65', 'name' => 'Сахалинская область'],
+            ['code' => '66', 'name' => 'Свердловская область'],
+            ['code' => '67', 'name' => 'Смоленская область'],
+            ['code' => '68', 'name' => 'Тамбовская область'],
+            ['code' => '69', 'name' => 'Тверская область'],
+            ['code' => '70', 'name' => 'Томская область'],
+            ['code' => '71', 'name' => 'Тульская область'],
+            ['code' => '72', 'name' => 'Тюменская область'],
+            ['code' => '73', 'name' => 'Ульяновская область'],
+            ['code' => '74', 'name' => 'Челябинская область'],
+            ['code' => '75', 'name' => 'Забайкальский край'],
+            ['code' => '76', 'name' => 'Ярославская область'],
+            ['code' => '77', 'name' => 'Москва'],
+            ['code' => '78', 'name' => 'Санкт-Петербург'],
+            ['code' => '79', 'name' => 'Еврейская автономная область'],
+            ['code' => '82', 'name' => 'Республика Крым'],
+            ['code' => '83', 'name' => 'Ненецкий автономный округ'],
+            ['code' => '86', 'name' => 'Ханты-Мансийский автономный округ - Югра'],
+            ['code' => '87', 'name' => 'Чукотский автономный округ'],
+            ['code' => '89', 'name' => 'Ямало-Ненецкий автономный округ'],
+            ['code' => '92', 'name' => 'Севастополь'],
+            ['code' => '95', 'name' => 'Чеченская Республика'],
+        ]);
+
+        Country::extend(function ($model) {
+            $model->setTable('winter_location_countries');
+        });
+
+        State::extend(function ($model) {
+            $model->setTable('winter_location_states');
+        });
```

Comment:
Why are you dynamically changing the table used? Once it gets to this point haven't the tables already been renamed?

Reply:
corrected